modular_skyrat/modules/gladiator/items.dm

```diff
@@ -0,0 +1,66 @@
+/obj/item/crusher_trophy/gladiator
+	name = "ashen bones"
+	desc = "A set of soot-coated ribs from a worthy warrior. Suitable as a trophy for a kinetic crusher."
+	icon_state = "demon_claws"
+	gender = PLURAL
+	denied_type = /obj/item/crusher_trophy/gladiator
+	bonus_value = 10
+
+/obj/item/crusher_trophy/gladiator/effect_desc()
+	return "the crusher to have a <b>[bonus_value]%</b> chance to block any incoming attack."
+
+/obj/item/crusher_trophy/gladiator/add_to(obj/item/kinetic_crusher/H, mob/living/user)
+	. = ..()
+	if(.)
+		H.block_chance += bonus_value
+
+/obj/item/crusher_trophy/gladiator/remove_from(obj/item/kinetic_crusher/H, mob/living/user)
+	. = ..()
+	if(.)
+		H.block_chance -= bonus_value
+
+/obj/item/gun/magic/staff/spellblade/agateram
+	name = "ancient blade agateram"
+	desc = "A millenia-old blade from a material that you can't even begin to fathom. It flows with the power of the Marked One who once held it." //That thing was too big to be called a sword. Too big, too thick, too heavy, and too rough, it was more like a large hunk of iron.
+	fire_sound = 'sound/magic/fireball.ogg'
+	ammo_type = /obj/item/ammo_casing/magic/spellblade
```

Comment:
spellblade
yea, no, not again, it was taken away for a reason.

Reply:
would you rather i set charges to 0 or something? it can't fit into any slot and can't be stored, and has less ammo than an actual spellblade + less armor piercing